openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -348,6 +348,9 @@ TCN_IMPLEMENT_CALL(jlong, SSLContext, make)(TCN_STDARGS, jint protocol, jint mod
 TCN_IMPLEMENT_CALL(jint, SSLContext, free)(TCN_STDARGS, jlong ctx)
```

Comment:
sgtm ... we can delay ... moving tcnative into netty's repo will make this easier later too